spec/EntitiesSpec.xml

```diff
@@ -338,74 +432,139 @@
+    GenerateWriterFunction="false">
+    <Field Name="definition_point_2" Code="13" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="13,23,33" />
+    <Field Name="definition_point_3" Code="14" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="14,24,34" />
+    <Field Name="definition_point_4" Code="15" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="15,25,35" />
+    <Field Name="definition_point_5" Code="16" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="16,26,36" />
   </Entity>
   <!-- ordinate -->
-  <Entity Name="OrdinateDimension" SubclassMarker="AcDbOrdinateDimension" TypeString="DIMENSION" BaseClass="DimensionBase" GenerateReaderFunction="false" GenerateWriterFunction="false">
-    <Field Name="definition_point_2" Code="13" Type="Point" DefaultValue="Point::origin()" CodeOverrides="13,23,33" />
-    <Field Name="definition_point_3" Code="14" Type="Point" DefaultValue="Point::origin()" CodeOverrides="14,24,34" />
+  <Entity Name="OrdinateDimension" SubclassMarker="AcDbOrdinateDimension" TypeString="DIMENSION"
+    BaseClass="DimensionBase" GenerateReaderFunction="false" GenerateWriterFunction="false">
+    <Field Name="definition_point_2" Code="13" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="13,23,33" />
+    <Field Name="definition_point_3" Code="14" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="14,24,34" />
   </Entity>
   <!--
 
   ELLIPSE
 
   -->
   <Entity Name="Ellipse" SubclassMarker="AcDbEllipse" TypeString="ELLIPSE" MinVersion="R13">
-    <Field Name="center" Code="10" Type="Point" DefaultValue="Point::origin()" CodeOverrides="10,20,30" />
-    <Field Name="major_axis" Code="11" Type="Vector" DefaultValue="Vector::x_axis()" CodeOverrides="11,21,31" />
-    <Field Name="normal" Code="210" Type="Vector" DefaultValue="Vector::z_axis()" DisableWritingDefault="true" CodeOverrides="210,220,230" />
+    <Field Name="center" Code="10" Type="Point" DefaultValue="Point::origin()"
+      CodeOverrides="10,20,30" />
+    <Field Name="major_axis" Code="11" Type="Vector" DefaultValue="Vector::x_axis()"
+      CodeOverrides="11,21,31" />
+    <Field Name="normal" Code="210" Type="Vector" DefaultValue="Vector::z_axis()"
+      DisableWritingDefault="true" CodeOverrides="210,220,230" />
     <Field Name="minor_axis_ratio" Code="40" Type="f64" DefaultValue="1.0" />
-    <Field Name="start_parameter" Code="41" Type="f64" DefaultValue="0.0" Comment="Ellipse start angle in radians." />
-    <Field Name="end_parameter" Code="42" Type="f64" DefaultValue="::std::f64::consts::PI * 2.0" Comment="Ellipse end angle in radians." />
+    <Field Name="start_parameter" Code="41" Type="f64" DefaultValue="0.0"
+      Comment="Ellipse start angle in radians." />
+    <Field Name="end_parameter" Code="42" Type="f64" DefaultValue="::std::f64::consts::PI * 2.0"
+      Comment="Ellipse end angle in radians." />
+  </Entity>
+  <!-- HATCH -->
```

Comment:
It would be nice to have a difference with what codes are left to be implemented?